lib/mpl_toolkits/mplot3d/axes3d.py

```diff
@@ -1606,13 +1639,8 @@ def _on_move(self, event):
 
             # update view
             vertical_axis = self._axis_names[self._vertical_axis]
-            self.view_init(
-                elev=elev,
-                azim=azim,
-                roll=roll,
-                vertical_axis=vertical_axis,
-                share=True,
-            )
+            self.view_init(elev, azim, roll, vertical_axis=vertical_axis,
```

Comment:
Not that I care too much, but why are we going back here to positional arguments?